ext/build_defaults.yaml

```diff
@@ -87,9 +87,9 @@ platform_repos:
   - name: sles-12-ppc64le
     repo_location: repos/sles/12/**/ppc64le
   - name: fedora-26-x86_64
-    repo_location: repos/fedora/26/**/x86_64
+    repo_location: repos/fedora/f26/**/x86_64
   - name: fedora-27-x86_64
-    repo_location: repos/fedora/27/**/x86_64
+    repo_location: repos/fedora/f27/**/x86_64
```

Comment:
I'm not really sure if you need these or not, since idk what you actually use these values for, but I don't *think* this affects the promote/compose issues at all.

Reply:
Hmm, it would be good to keep the non-f versions -- I'll update these and re-test with fedora